test/rtos_drivers/wifi/check_wifi.sh

```diff
@@ -73,5 +73,3 @@ echo "****************"
 echo "* Parse Result *"
 echo "****************"
 python ${REPO_ROOT}/test/rtos_drivers/python/parse_test_output.py testing/test.rpt -outfile="testing/test_results" --print_test_results --verbose
-
-pytest ${REPO_ROOT}/test/rtos_drivers/wifi
```

Comment:
Is this test performed elsewhere?  If not, why has it been removed?

Reply:
This test was moved in [PR-192](https://github.com/xmos/fwk_rtos/pull/192) and now it is run from the [Jenkinsfile](https://github.com/xmos/fwk_rtos/blob/develop/Jenkinsfile#L117).